src/vue/modules/create-asset-sale/store/types.js

```diff
@@ -0,0 +1,19 @@
+const mutations = {
+  SET_BALANCES: 'SET_BALANCES',
+}
+
+const actions = {
+  LOAD_KV_KYC_REQUIRED: 'LOAD_KV_KYC_REQUIRED',
+  LOAD_BLOB_ID: 'LOAD_BLOB_ID',
+  LOAD_ASSETS: 'LOAD_ASSETS',
```

Comment:
You have this action, but don't have any mutation or getter. Clean up this file

Reply:
ok